src/app/views.py

```diff
@@ -103,6 +105,24 @@ def generateLicenseXml(licenseOsi, licenseIdentifier, licenseName, licenseSource
     xmlString = ET.tostring(root, encoding='utf8', method='xml')
     return xmlString
 
+def createIssue(licenseName, licenseIdentifier, licenseSourceUrls, licenseOsi):
+    """ View for creating an GitbHub issue
+    when submitting a new license request
+    """
+    myToken = ''
```

Comment:
We should move the token to an external file which is not under source control for security.  There are a few methods we can use to do this.  My preference would be to create a separate Python file with myToken defined.  We would import from this file here.  Then we would add the file to the .gitignore so that the tokens do not get accidentally committed.

Reply:
That solution sounds good. I will work on it.